File: R/8kyu/8kyu-get-nth-even-number.r
# Get Nth Even Number
# Language: R
# Version 3.4.1
# Difficulty: 8 kyu

nth_even <- function(n) {
  n * 2 - 2
}


# Tests
test_that("Fixed tests", {
  expect_equal(nth_even(1), 0)
  expect_equal(nth_even(2), 2)
  expect_equal(nth_even(3), 4)
  expect_equal(nth_even(100), 198)
  expect_equal(nth_even(1298734), 2597466)
})

File: Julia/8kyu/8kyu-get-nth-even-number.jl
# Get Nth Even Number
# Language: Julia
# Version: 1.5
# Difficulty: 8 kyu

function ntheven(n)
  n * 2 - 2
end


# Tests
using FactCheck
facts("Fixed tests") do
  @fact ntheven(1) --> 0
  @fact ntheven(2) --> 2
  @fact ntheven(3) --> 4
  @fact ntheven(100) --> 198
  @fact ntheven(1298734) --> 2597466
end